test/wdio/slot-reorder/cmp.test.tsx

```diff
@@ -131,21 +131,21 @@ describe('slot-reorder', () => {
     ordered();
 
     await $('button').click();
-    await browser.waitUntil(() => {
+    await browser.waitUntil(async () => {
       return document.querySelector('div.reordered');
     });
 
     reordered();
 
     await $('button').click();
-    await browser.waitUntil(() => {
+    await browser.waitUntil(async () => {
       return !document.querySelector('div.reordered');
     });
 
     ordered();
 
     await $('button').click();
-    await browser.waitUntil(() => {
+    await browser.waitUntil(async () => {
```

Comment:
Does passing an `async` function to `waitUntil` do anything special? Or do we not need these `async` fns?

Reply:
I've seen race conditions saying `.then is not a function` and my assumptions are that this has to do with WebdriverIO. It is on the list of things I'ld like to look into eventually. In the meeting, I am adding this to avoid the race conditions.